src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -23,7 +23,7 @@
             [puppetlabs.puppetdb.reports :as reports]
             [puppetlabs.puppetdb.facts :as facts :refer [facts-schema]]
             [puppetlabs.kitchensink.core :as kitchensink]
-            [puppetlabs.puppetdb.scf.storage-utils :as scf-utils]
+            [puppetlabs.puppetdb.scf.storage-utils :as sutils]
```

Comment:
Hahaha, yeah, I see that you started thinking like me at some point in this patch :-).